modules/installing-ocp-agent-ibm-z-zvm.adoc

```diff
@@ -29,10 +29,10 @@ ignition.firstboot ignition.platform.id=metal \
 console=tty1 console=ttyS1,115200n8 \
 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"
 ----
-<1> For the `coreos.live.rootfs_url=` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are booting. Only HTTP and HTTPS protocols are supported.
-<2> For the `ip=` parameter, assign the IP address automatically using DHCP, or manually assign the IP address, as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
-<3> The default is '1'. Omit this entry when using an OSA network adapter.
-<4> For installations on DASD-type disks, use `rd.dasd=` to specify the DASD where {op-system} is to be installed. Omit this entry for FCP-type disks.
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip` parameter, assign the IP address automatically using DHCP, or manually assign the IP address, as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
```

Comment:
Worthwhile adding a link to this doc in the provider Ad resources section for your doc?

Reply:
Expand [DHCP](DHCP) for this module that might be reusable?

Reply:
Sorry, could you clarify what you mean in your second comment? The link doesn't seem to take me anywhere. Do you mean spell DHCP out? IMO I think DHCP is a pretty well-known networking term like IP, TCP, and HTTPS, in which case our style guide gives us the leeway to not spell it out.

But that's just my opinion, I can spell it out if you'd like.

Reply:
Hi @skopacz1 . Please ignore the formatting that should not have been formatted. Yeah, knowing what is common industry terms is difficult as it's suubjective. Probably easier to spell it out but I'll leave that as your decision :smile:

Reply:
I agree DHCP, OSA, and DASD are commonly used and well known to IBM Z customers. The customer already has all those set up on his IBM Z machine before he starts installing OCP.

Reply:
Thanks for confirming. I do not know IBM products enought to know their common lingo. LGTM!